### PR TITLE
Move Polishs docs to Image Optimization

### PR DESCRIPTION
**Pull request by marciocloudflare**
adds content to images and removes from cache

### CODE REVIEW

**Review by sejoker — APPROVED** [review]

**Review by kornelski — APPROVED** [review]

File: products/images/src/content/faq.md [hunk 1]
```diff
@@ -105,4 +106,12 @@ Refer to our [Plans](https://www.cloudflare.com/plans/) page for up-to-date info
 
 ### Resizing failed and I received an error response with a code. What does it mean?
 
-Visit our [Troubleshoot Image Resizing problems](https://support.cloudflare.com/hc/en-us/articles/4412024022029-Troubleshoot-Image-Resizing-problems) page were we have a troubleshoot guide for some of the more common situations, including error responses.
+Visit our [Troubleshoot Image Resizing problems](https://support.cloudflare.com/hc/articles/4412024022029) page were we have a troubleshoot guide for some of the more common situations, including error responses.
```
Comment (pedrosousa) [inline products/images/src/content/faq.md]:
```suggestion
Refer to [Troubleshoot Image Resizing problems](https://support.cloudflare.com/hc/articles/4412024022029) for more information on how to troubleshoot some of the more common issues, including error responses.
```

File: products/images/src/content/faq.md [hunk 2]
```diff
@@ -105,4 +106,12 @@ Refer to our [Plans](https://www.cloudflare.com/plans/) page for up-to-date info
...
+
+---
+
+## Polish FAQ
+
+### How can I troubleshoot common `Cf-Polished` statuses?
+
+Visit our [Troubleshoot common Cf-Polished statuses](https://support.cloudflare.com/hc/articles/4412219637389) page were you can find a list of common `Cf-Polished` statuses and how to troubleshoot them.
```
Comment (pedrosousa) [inline products/images/src/content/faq.md]:
```suggestion
Refer to the [Troubleshoot common Cf-Polished statuses](https://support.cloudflare.com/hc/articles/4412219637389) page were you can find a list of common `Cf-Polished` statuses and how to troubleshoot them.
```

File: products/images/src/content/index.md [hunk 3]
```diff
@@ -6,10 +6,12 @@ pcx-content-type: overview
...
 # Cloudflare Image Optimization
 
-Choose between Cloudflare Images and Cloudflare Image Resizing, two products tailored to your different needs.
+Choose between Cloudflare Images, Cloudflare Image Resizing, and Polish, three products tailored to your different needs.
 
 With Cloudflare Images you can build a scalable image pipeline on Cloudflare servers, to store, resize, optimize, and deliver images in a fast and secure manner.
 
 With Image Resizing you can use Cloudflare’s edge platform to resize, adjust quality, and convert images to WebP or AVIF format on demand, independently of where they are stored.
 
+Cloudflare Polish is a one-click image optimization product, that automatically optimizes images in your site.
```
Comment (pedrosousa) [inline products/images/src/content/index.md]:
```suggestion
Cloudflare Polish is a one-click image optimization product that automatically optimizes images in your site.
```

File: products/images/src/content/polish/activate-polish.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+Images in the [cache must be purged](https://developers.cloudflare.com/cache/how-to/purge-cache) or expired before seeing any changes in Polish settings.
+
+<Aside type="warning" header="Warning">
+
+Do not activate Polish and Image Resizing simultaneously. Image Resizing already applies lossy compression, which makes Polish redundant.
+
+</Aside>
+
+1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com/) and select the account for the domain where you want to activate Polish.
```
Comment (pedrosousa) [inline products/images/src/content/polish/activate-polish.md]:
```suggestion
1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com/) and select the account and domain where you want to activate Polish.
```
I believe you have to select both an account and a domain.

File: products/images/src/content/polish/activate-polish.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+1. Click **Speed** > **Optimization**.
+1. Under **Polish**, select *Lossy* or *Lossless* from the dropdown menu.
```
Comment (pedrosousa) [inline products/images/src/content/polish/activate-polish.md]:
```suggestion
1. Under **Polish**, select *Lossy* or *Lossless* from the drop-down menu.
```

File: products/images/src/content/polish/activate-polish.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+1. (Optional) Click **WebP**. Enable this option if you want to further optimize PNG and JPEG images stored in the origin server, and serve them as WebP files to browser that support this format.
```
Comment (pedrosousa) [inline products/images/src/content/polish/activate-polish.md]:
```suggestion
1. (Optional) Click **WebP**. Enable this option if you want to further optimize PNG and JPEG images stored in the origin server, and serve them as WebP files to browsers that support this format.
```

File: products/images/src/content/polish/activate-polish.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+
+<Aside type="note">
+
+To ensure WebP is not served from cache to a browser lacking WebP support, disable WebP at your origin web server when using Polish.
```
Comment (pedrosousa) [inline products/images/src/content/polish/activate-polish.md]:
```suggestion
To ensure WebP is not served from cache to a browser without WebP support, disable WebP at your origin web server when using Polish.
```

File: products/images/src/content/polish/index.md [hunk 8]
```diff
@@ -0,0 +1,38 @@
+---
+order: 3
+pcx-content-type: concept
+---
+
+# Cloudflare Polish
+
+Cloudflare Polish is a one-click image optimization product, that automatically optimizes images in your site. Polish strips metadata from images, and reduces image size through lossy or lossless compression, to accelerate the speed of image downloads.
```
Comment (pedrosousa) [inline products/images/src/content/polish/index.md]:
```suggestion
Cloudflare Polish is a one-click image optimization product that automatically optimizes images in your site. Polish strips metadata from images and reduces image size through lossy or lossless compression to accelerate the speed of image downloads.
```

File: products/images/src/content/polish/index.md [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+
+## Polish compression options
+
+### Lossless 
+
+Lossless attempts to strip most metadata, like EXIF data, but does not change the image detail. Effectively, when uncompressed, a lossless image is identical to the original. On average, Lossless compression reduces file size by 21%.
+
+### Lossy
+
+Lossy attempts to strip most metadata and compresses images by approximately 15 percent. When uncompressed, some of the redundant information from the original image is lost. On average, using Lossy mode reduces file size by 48%. Lossy has the same effect as Lossless when applied to PNG.
```
Comment (pedrosousa) [inline products/images/src/content/polish/index.md]: Considering using either "percent" or "%" (wherever possible).

**Review by pedrosousa — APPROVED** [review]

File: products/images/src/content/polish/index.md (hunk 9 above)
Comment (marciocloudflare) [reply products/images/src/content/polish/index.md]: Done

**Pull request merged**